pkg/csi/service/common/vsphereutil.go

```diff
@@ -1057,6 +1057,34 @@ func DeleteSnapshotUtil(ctx context.Context, manager *Manager, csiSnapshotID str
 	return nil
 }
 
+// GetCnsVolumeType is the helper function that determines the volume type based on the volume-id
+func GetCnsVolumeType(ctx context.Context, manager *Manager, volumeId string) (string, error) {
```

Comment:
Isn't there an existing utils function checking volume type?

Reply:
I looked for it, couldn't find it.